packages/config/config/devices/0x010f/fgms001.json

```diff
@@ -487,7 +529,7 @@
 			"#": "42",
 			"$if": "firmwareVersion >= 3.2",
 			"label": "Illuminance report - interval",
-			"description": "Time interval between consecutive illumination reports",
+			"description": "Time interval (seconds) between consecutive illumination reports (sent even when there is no change).",
```

Comment:
The unit should be specified using the `"unit"` property, not as part of the description.